views/index.html.twig

```diff
@@ -34,8 +34,8 @@
             {% if name %}{{ name }}{% else %}Composer repository{% endif %}
         </h1>
 
-        <small id="lastupdate">
-            Last updated: <time datetime="{{ "now"|date(constant('\DateTime::ISO8601')) }}">{{ "now"|date(constant('\DateTime::RSS')) }}</time>
+        <small>
+            Last updated: <time datetime="{{ "now"|date(constant('\DateTime::RFC3339')) }}">{{ "now"|date(constant('\DateTime::RFC3339')) }}</time>
```

Comment:
Ok, I would argue that `DATE_COOKIE` is more readable than `DATE_ISO8601` though. What is your thought on this?

Also, as far as I understand it, this puts the "format" in the tag, not the value (e.g. `l, d-M-Y H:i:s T` in the case of `COOKIE`).